### ISSUE TITLE
Error after login when using Domains

### ISSUE THREAD

**Issue by sourcex** [Ylianst/MeshCentral#163]
I'm attempting to enable some additional tenants on my Ubuntu 18.04 server running 0.3.1-r.  After editing the config and creating an account, after login I receive this message:

> TypeError: First argument must be a string, Buffer, ArrayBuffer, Array, or array-like object.
>     at Function.Buffer.from (buffer.js:183:11)
>     at handleRootRequestEx (/home/totallyausername/node_modules/meshcentral/webserver.js:1191:629)
>     at handleRootRequest (/home/totallyausername/node_modules/meshcentral/webserver.js:1048:13)
>     at Layer.handle [as handle_request] (/home/totallyausername/node_modules/express/lib/router/layer.js:95:5)
>     at next (/home/totallyausername/node_modules/express/lib/router/route.js:137:13)
>     at Route.dispatch (/home/totallyausername/node_modules/express/lib/router/route.js:112:3)
>     at Layer.handle [as handle_request] (/home/totallyausername/node_modules/express/lib/router/layer.js:95:5)
>     at /home/totallyausername/node_modules/express/lib/router/index.js:281:22
>     at Function.process_params (/home/totallyausername/node_modules/express/lib/router/index.js:335:12)
>     at next (/home/totallyausername/node_modules/express/lib/router/index.js:275:10)
>     at SendStream.error (/home/totallyausername/node_modules/serve-static/index.js:121:7)
>     at emitOne (events.js:116:13)
>     at SendStream.emit (events.js:211:7)
>     at SendStream.error (/home/totallyausername/node_modules/send/index.js:270:17)
>     at SendStream.onStatError (/home/totallyausername/node_modules/send/index.js:421:12)
>     at next (/home/totallyausername/node_modules/send/index.js:764:28)

This is the slightly edited domains section of my config.
```
 "domains": {
    "": {
      "title": My MeshCentral",
      "_title2": "",
      "_userQuota": 1048576,
      "_meshQuota": 248576,
      "newAccounts": 1,
      "_footer": "<a href='https://twitter.com/totallyme'>Twitter</a>"
    },
    "cust1": {
      "title": "CUST1 MeshCentral",
      "newAccounts": 1
    },
    "cust2": {
        "title": "CUST2 MeshCentral",
        "newAccounts": 1
    }
  },
```

I've tested on Chrome and Firefox.

**Comment by Ylianst** [comment]
Thanks for the detailed report. I am on vacation and flying back tomorrow evening, will be in the office Thursday to start fixing things. Certainly interested to look at what is going on here.

**Comment by Ylianst** [comment]
Thanks for reporting this. Should be fixed in MeshCentral v0.3.1-s. Let me know if it works for you.

**Comment by sourcex** [comment]
I no longer receive the error when logging in and can continue testing.  Thank you.
